app/src/ts/upload.ts

```diff
@@ -3,8 +3,8 @@
 // The script for uploading a file
 
 import * as $ from "jquery";
+const remote = require('electron').remote;
```

Comment:
Is this the best way to import this? Is there no way to use `import { } from 'electron'` syntax?